site/content/blog/2019-04-16-svelte-for-new-developers.md

```diff
@@ -56,52 +56,43 @@ To write code, you need a good editor. The most popular choice is [Visual Studio
 
-In the `package.json` file, there is a section called `"scripts"`. These scripts define shortcuts for working with your application — `dev`, `build` and `start`. To launch your app in development mode, type the following:
+In the `package.json` file, there is a section called `"scripts"`. These scripts define shortcuts for working with your application — `dev`, `build` and `preview`. To launch your app in development mode, type the following:
 
 ```bash
 npm run dev
 ```
 
-Running the `dev` script starts a program called [Rollup](https://rollupjs.org/guide/en/). Rollup's job is to take your application's source files (so far, just `src/main.js` and `src/App.svelte`), pass them to other programs (including Svelte, in our case) and convert them into the code that will actually run when you open the application in a browser.
+Running the `dev` script starts a program called [Vite](https://vitejs.dev/). Vite's job is to take your application's source files, pass them to other programs (including Svelte, in our case) and convert them into the code that will actually run when you open the application in a browser.
 
-Speaking of which, open a browser and navigate to http://localhost:8080. This is your application running on a local *web server* (hence 'localhost') on port 8080.
+Speaking of which, open a browser and navigate to http://localhost:3000. This is your application running on a local *web server* (hence 'localhost') on port 3000.
 
-Try changing `src/App.svelte` and saving it. The application will reload with your changes.
+Try changing `src/App.svelte` and saving it. The application will update with your changes.
 
 
 ## Building your app
 
-In the last step, we were running the app in 'development mode'. In dev mode, Svelte adds extra code that helps with debugging, and Rollup skips the final step where your app's JavaScript is compressed using [Terser](https://terser.org/).
+In the last step, we were running the app in 'development mode'. In dev mode, Svelte adds extra code that helps with debugging, and Vite skips the final step where your app's JavaScript is compressed.
 
 When you share your app with the world, you want to build it in 'production mode', so that it's as small and efficient as possible for end users. To do that, use the `build` command:
 
 ```bash
 npm run build
 ```
 
-Your `public` directory now contains a compressed `bundle.js` file containing your app's JavaScript. You can run it like so:
+Your `dist` directory now contains an optimised version of your app. You can run it like so:
 
 ```bash
-npm run start
+npm run preview
```

Comment:
This serves your app on 4173, unlike dev's 3000. If that's what we want to go with because it's Vite's default, then maybe it could be mentioned here. Or if we don't want to rely on that consistently being the port, we could just document that the CLI outputs the port it's serving on.

Reply:
I was sort of hoping it would be self-explanatory — is that too much to hope for?

Reply:
I think the Vite CLI will print the port so I don't know that we need much documentation around it